src/Tag/Base/BooleanInputTag.php

```diff
@@ -36,12 +36,18 @@ final public function checked(bool $checked = true): static
      *
      * @param string|null $label Input label.
      * @param array $attributes Name-value set of label attributes.
+     * @param bool $wrap Whether to wrap input with label tag. If set to `false`, label will be rendered side on side
```

Comment:
What do you mean by "side on side input"?

Reply:
fixed